src/zcbenchmarks.cpp

```diff
@@ -88,3 +90,15 @@ double benchmark_create_joinsplit()
     assert(pourtx.Verify(*pzerocashParams));
     return ret;
 }
+
+double benchmark_solve_equihash()
+{
+    const char *testing = "testing";
```

Comment:
If we want to compare the performance of old code to new code, then should we mine a block or call `BasicSolve`? My thinking was that `BasicSolve` would have relatively constant performance, and mining a block would have a lot of variance because of the possibility of not finding a solution, and so the benchmark wouldn't be able to pick up on small-ish changes in performance. This is only the case if `BasicSolve` doesn't change how often it finds a solution (which I think it won't, since either there is a solution for the input, in which case it finds it, or there isn't.) If we know the probability of a solution existing we can extrapolate to the time it'll take to mine a block. Or, we can have *both* because I can just add another benchmark that mines a block, and keep the one that calls `BasicSolve`. What do you think?